perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -566,6 +566,19 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	},
 
 	/*#
+	 *
```

Comment:
Missing javadoc for RPC - might fail the script parsing it. Either fill javadoc or remove java comment.